docs/GettingStarted.md

```diff
@@ -119,23 +141,126 @@ Check the properties of the created stream. The **format** is one property which
     oboe::AudioFormat format = stream->getFormat();
     LOGI("AudioStream format is %s", oboe::convertToText(format));
 
-Now start the stream. 
+Now start the stream.
 
     stream->requestStart();
 
 At this point you should start receiving callbacks.
 
 ### Closing the stream
-It is important to close your stream when you're not using it to avoid hogging audio resources which other apps could use. To do this use: 
+It is important to close your stream when you're not using it to avoid hogging audio resources which other apps could use. To do this use:
 
     stream->close();
 
 `close()` is a blocking call which also stops the stream.
 
 It is usually advisable to close your stream when [`Activity.onPause()`](https://developer.android.com/guide/components/activities/activity-lifecycle#onpause) is called.
```

Comment:
This is particularly true when using SharingMode::Exclusive because you might prevent other apps from using the MMAP data path.